src/i18n/nl_NL/buildings.ts

```diff
@@ -26,6 +26,20 @@ export default {
                 givesParkingLots: 0,
                 unlocksVehicleTypes: [16, 30, 52, 63],
             },
+            ...multiplyExtension(
+                {
+                    caption: 'Haakarmbak parkeerplaats',
+                    credits: 100_000,
+                    coins: 20,
+                    duration: '7 Dagen',
+                    isVehicleExtension: true,
+                    givesParkingLots: 1,
+                    unlocksVehicleTypes: [26, 27, 29, 32, 45, 51, 61],
+                    parkingLotReservations: [[27, 29, 32, 45, 51, 61]],
```

Comment:
Ist ID 26 the default vehicle? Eg. the german FuStW?

Reply:
These are the Abrollcontainer parking lots. 26 is the WLF vehicle which is unlocked with the first Abrollcontainer extension on NL.